Services/DataBaseService/DbModels/DataBaseContext.cs

```diff
@@ -0,0 +1,31 @@
+﻿using DataBaseService.DbModels;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System.Reflection;
+
+namespace DataBaseService
+{
+    public class DataBaseContext: DbContext
+    {
+        public DbSet<DbUser> Users { get; set; }
+
+        private readonly IConfiguration configuration;
```

Comment:
I thought we agreed to right constructor after fields and properties

Reply:
Replace this stuff